cli/command/system/prune.go

```diff
@@ -67,6 +67,14 @@ func runBuildCachePrune(dockerCli command.Cli, _ opts.FilterOpt) (uint64, string
 }
 
 func runPrune(dockerCli command.Cli, options pruneOptions) error {
+
+	progressChan := pruneProgressChan{
+		errorChan:         make(chan error),
+		isEventsListening: make(chan bool),
+	}
+
+	go runPruneProgress(dockerCli, &progressChan, options)
```

Comment:
`runPruneProgress` starts a couple go routines, so it seems like it doesn't really need to be run in a goroutine as well. Would it be fine to run `countObjectsToRemove()` synchronously ?

Reply:
Yep, corrected.